<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Youtube Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="gb_inputs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Inputs</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>URL:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_url"/>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="le_output"/>
         </item>
         <item>
          <widget class="QToolButton" name="btn_directory">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_watch">
           <property name="text">
            <string>Watch</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_download">
           <property name="text">
            <string>Download</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Directory output:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="rb_all">
            <property name="text">
             <string>All</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_144">
            <property name="text">
             <string>144p</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_240">
            <property name="text">
             <string>240p</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_360">
            <property name="text">
             <string>360p</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_480">
            <property name="text">
             <string>480p</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_720">
            <property name="text">
             <string>720p</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_1080">
            <property name="text">
             <string>1080p</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_audio">
            <property name="text">
             <string>Audio Only</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="sw_player">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
